app/manager/payroll-records-table/page.tsx

```
@@ -97,6 +99,16 @@ export default function PayrollTable() {
                 </div>
             </CardHeader>
             <CardContent>
+                {isLoading && (
+                    <p role="status" aria-live="polite" className="text-sm text-muted-foreground mb-4">
```

Biome (2.4.6)

[error] The elements with this role can be changed to the following elements:
<output>

(lint/a11y/useSemanticElements)